packages/http-client-csharp/eng/pipeline/publish.yml

```diff
@@ -97,15 +97,14 @@ extends:
                 displayName: Install tsp-client
 
               - task: PowerShell@2
-                displayName: Update package.json and generate emitter-package.json files
+                displayName: Update package.json with injected dependencies
```

Comment:
nit: should we still include that we are generating the emitter artifacts in the display name?